doctor/views.py

```
@@ -0,0 +1,339 @@
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework import status
+from django.contrib.auth import get_user_model
+import logging
+
+from .serializers import (
+    DoctorProfileSerializer,
+    DoctorProfileCreateSerializer,
+    DoctorScheduleSerializer,
+    DoctorCertificateSerializer,
```

Ruff (0.12.2)

`.serializers.DoctorCertificateSerializer` imported but unused

Remove unused import

(F401)